cmd/server/main.go

```diff
@@ -26,6 +26,16 @@ func main() {
 			Value: "/var/lib/ocid/sandbox",
 			Usage: "ocid pod sandbox dir",
 		},
+		cli.Float64Flag{
```

Comment:
We don't need to pass this from CLI. It should come as part of container configuration.